analyzer/codechecker_analyzer/buildlog/build_manager.py

```diff
@@ -76,18 +76,8 @@ def perform_build_command(logfile, command, context, keep_link, silent=False,
                     'using a current copy for logging.')
         original_env = os.environ.copy()
 
-    # Run user's commands with intercept.
-    if host_check.check_intercept(original_env):
-        LOG.info("Using intercept-build.")
-        final_command = command
-        command = ' '.join(["intercept-build",
-                            "--cdb", logfile,
-                            "sh -c", shlex.quote(final_command)])
-        log_env = original_env
-        LOG.debug_analyzer(command)
-
-    # Run user's commands in shell.
-    else:
+    # Run user's commands in shell, and preload ldlogger.
+    if host_check.check_ldlogger(original_env):
```

Comment:
The original_env doesn't contain environment variable which are set by the top level script and using this function. For this reason use the current environment:
```suggestion
    if host_check.check_ldlogger(os.environ):
```